conda-recipe/run_test.sh

```diff
@@ -0,0 +1 @@
+nosetests -v metadatastore --cover-package metadatastore --with-coverage
```

Comment:
Sorry if I seem to be pushing back on this too hard, but I've never seen another project do this and debugging Travis can eat a lot of time as we know. What particular situation are you trying to catch? Where the `.travis.yml` and the `meta.yml` are out of sync, with respect to versions of deps?